apps/heureka/src/components/issueMatches/IssueMatchesDetails.jsx

```diff
@@ -36,6 +36,13 @@ const IssueMatchesDetails = () => {
               <LoadElement elem={issue?.issue?.primaryName} />
             </DataGridCell>
           </DataGridRow>
+          <DataGridRow>
+            <DataGridHeadCell>Description</DataGridHeadCell>
+            <DataGridCell>
+              {/* // Take description from the first issueVariant, because if there are multiple, they have the same priority (edge case). */}
+              <LoadElement elem={issue?.effectiveIssueVariants?.edges?.[0]?.node?.description} />
```

Comment:
Could yo not extract this one in a variable above that represent a proper name and delete then a comment please?